module-types/LiteOn/CM-12CP-010U.yaml

```diff
@@ -0,0 +1,34 @@
+---
+manufacturer: LiteOn
+model: CM-12CP-010U
+part_number: CM-12CP-010U
+comments: |
+  LiteOn Power Shelf Controller
+
+  * Monitors up to 10 PSUs and 8 BBUs
+  * Hot-swappable for easy installation and maintenance
+  * Configurable alarm and reminder notifications
+  * Real-time monitoring capabilities for power shelves and battery backup system
+  * Real-time alarm logging and delivery
+  * SNMP (v2c, v3), Web server, RestAPI, and IPMI support for remote management
+  * Secure web interface
+  * Efficiency management system for optimizing power shelf efficiency
+  * 10/100/1000 Ethernet port support IPv6, SNMPv3
+  * CAN communication port
+weight: 1.59
+weight_unit: lb
+console-ports:
+  - name: serial
+    label: SERVICE
+    type: de-9
+  - name: can-1a
+    label: CAN-1A
+    type: rj-45
+  - name: can-1b
+    label: CAN-1B
+    type: rj-45
+interfaces:
+  - name: mgmt
+    label: LAN
+    type: 1000base-t
+    mgmt_only: true
```

Comment:
I am aware of the {module} naming structure. I'm still waiting on some details regarding how these actually show up in the OS, but I named them like this because there is only one slot for this type of module, and I'm not entirely sure if the manufacturer has any devices that would even support an additional module of this type.

Reply:
Let me know whenever you get this wrapped up so I can re-review this PR

Reply:
I never did receive the information needed to confirm or deny the existence of one of these units that can handle more than one 'power supply controller', so I have left this as a hard-coded name for now. If you would prefer that I use `mgmt{module}` I can do that as well.